README.md

```diff
@@ -31,4 +31,10 @@ npm start
 runs the app in the development mode.\
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits and you will also see any lint errors in the console.
\ No newline at end of file
+The page will reload if you make edits and you will also see any lint errors in the console.
+
+## Notes on web packing and deploying
+
+* Rive's web runtime uses Wasm to create a small, performant engine. Not all web packers play nicely with Wasm; if you run into difficulties, check your pipeline's documentation. In this example, we load the Wasm file from the *unpkg* cdn, but could just as easily load it from your hosting environment. Take a look at the [RiveLoader.js](https://github.com/alxgibsn/rive-examples-react/blob/main/src/RiveLoader.js) source file. 
```

Comment:
This is great, but I'd change this to say web *bundlers* instead of web packers to not confuse with webpack specifically.

Reply:
Good catch; let me update that.